### ISSUE TITLE
Ruby gem

### ISSUE THREAD

**Issue by mermop** [GOVTNZ/govtnz-design-system#23]
Hello! It would be great if there was support for Ruby on Rails applications through a gem that supported the asset pipeline.

I'm working on Ruby on Rails applications at MSD and the Service Innovation Lab and it would be a lot easier to incorporate this design system to existing applications (that don't load any CSS through the JS ecosystem) if there was a Ruby gem. 

Examples include:
- https://github.com/twbs/bootstrap-rubygem - the Rails engine is at https://github.com/twbs/bootstrap-rubygem/blob/master/lib/bootstrap/engine.rb and can effectively be lifted wholesale
- https://github.com/zurb/foundation-rails
- https://github.com/thoughtbot/bourbon

**Comment by holloway** [comment]
Good idea, I'll propose that to the product owner and leave this issue open so we can track it.
